docs/02-Scales.md

```diff
@@ -310,14 +310,15 @@ The following additional configuration options are provided by the radial linear
 
 Name | Type | Default | Description
 --- | --- | --- | ---
-lineArc | Boolean | false | If true, circular arcs are used else straight lines are used. The former is used by the polar area chart and the latter by the radar chart
+labelPoints | Boolean | true | If true, angle lines and point labels are drawn.
```

Comment:
Is it something equivalent to `pointLabels.display && angleLines.display`. If so, maybe it would be better to split this option in these two `*.display` for a finer control. Else, I don't think the option name is great because then this scale will have very similar option names:  `labelPoints` and `pointLabels`, which is confusing.